### PR TITLE
Added category sorting

### PR DESCRIPTION
**Pull request by gerazov**
Another try at addressing #128

### CODE REVIEW

**Comment by kristijanhusak** [comment]
Please first confirm that current sorting is not behaving the same as in Emacs orgmode.

File: lua/orgmode/agenda/views/agenda.lua [hunk 1]
```diff
@@ -37,6 +57,12 @@ local function sort_agenda_items(agenda_items)
       return a.headline:get_priority_sort_value() > b.headline:get_priority_sort_value()
     end
 
+    -- if both are on the same day check category
```
Comment (gerazov) [inline lua/orgmode/agenda/views/agenda.lua:60]: @kristijanhusak can you help with this part - basically I want to give preference to overdue items over categories, but it doesn't seem to do the trick ...

Comment (gerazov) [reply lua/orgmode/agenda/views/agenda.lua:60]: Ok - figured it out - it's `headline_date:is_same()` rather than `date:is_same()`

**Comment by gerazov** [comment]
ok now it works with deadlines as well https://github.com/nvim-orgmode/orgmode/issues/128#issuecomment-997000672

**Pull request closed without merging**